### PR TITLE
Update Token module endpoints - Closes #7579 #7660

### LINKED ISSUE #7579: Update token module endpoints
### Description

1. Add token module endpoint `getInitializationFees` which returns below using config
```
{
  "userAccount": "50000000",
  "escrowAccount": "50000000",
}
```

2. Update token module endpoints to use updated stores referring to [LIP stores](https://github.com/LiskHQ/lips/blob/98eafb761a4792cc0918996a13f72703629195ff/proposals/lip-0051.md#token-module-store)
3. Add token module endpoint `isSupported`
req
```
{
  tokenID: '',
}
```
res
```
{
  supported: true
}
```
4. Add token module endpoint `getSupportedTokens`
res
```
  supportedTokens: []
```
where array should include `['*']` when all tokens are supported. It should also include mainchain / native tokens.
If all native token from a chain is supported, it should return `['00000020********']`

### Acceptance Criteria

- It should be reflected in the metadata
- Unit test for the endpoint is added

### Additional Information
- This change requires #7573 and #7574 for the config values

### PR DESCRIPTION
**Pull request by bobanm**
### What was the problem?

This PR resolves #7579 #7660

### How was it solved?

* Fixed `TokenModule.init()` to get config from a `ModuleConfig` object
* Implemented `isSupported` endpoint
* Implemented `getInitializationFees` endpoint
* Partially implemented `getSupportedTokens` endpoint

### How was it tested?

Added unit tests. Skipped the one that should return false when checking if a not-supported token is supported.

### CODE REVIEW

File: framework/src/modules/token/module.ts [hunk 1]
```diff
@@ -163,29 +171,30 @@ export class TokenModule extends BaseInteroperableModule {
...
 		this.stores.get(SupportedTokensStore).registerOwnChainID(this._ownChainID);
 		this.crossChainTransferCommand.init({ ownChainID: this._ownChainID });
 
-		this.method.init({
-			ownChainID: this._ownChainID,
-			escrowAccountInitializationFee: BigInt('50000000'),
-			feeTokenID: Buffer.from(config.feeTokenID, 'hex'),
-			userAccountInitializationFee: BigInt('50000000'),
-		});
+		this.method.init(Object.assign(config, { ownChainID: this._ownChainID }));
```
Comment (shuse2) [inline framework/src/modules/token/module.ts:191]: you can use spread operator instead of `Object.assign`

**Review by shuse2 — COMMENTED** [review]
LGTM for the existing implementation

**Review by mosmartin — APPROVED** [review]
🚀

File: framework/src/modules/token/module.ts (hunk 1 above)
Comment (bobanm) [reply framework/src/modules/token/module.ts]: Good point. Updated.

**Review by shuse2 — APPROVED** [review]

**Pull request merged**